Newsletter: 2024-09 (#43)
OBF/newsletter: newsletters/2024-09.md

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2024-03.md)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

# OBF Newsletter September 2024

TL;DR: Watch the BOSC 2024 recordings, save the date for BOSC 2025, share or apply for the upcoming OBF event fellowship application and join the Pydna hackathon!

In longer form: 

## OBF News

### BOSC news

#### BOSC 2024: Watch the videos!

> This year we celebrated the 25th annual _Bioinformatics Open Source Conference_ (BOSC 2024), 

Reviewer: ```suggestion
This year we celebrated the 25th annual _Bioinformatics Open Source Conference_ ([BOSC 2024](https://www.open-bio.org/events/bosc-2024/)), 
```

which took place in July as part of ISMB 2024 alongside the post-conference _CollaborationFest_, 
a collaborative work event (including but not limited to hacking).

> Our amazing conference organising committee has [written up a report for F1000 about the conference](https://docs.google.com/document/d/1P8Vzr_Rja7-yS686cL9TX7amoUfL5jt_qyH1wu3JGtw/edit),

Reviewer: I don't want this to go out with the raw gdoc; we will hopefully have an F1000 URL before this newsletter goes out.

Author: Given that the one community item we have is somewhat time critical, I'd like this to go out by the end of the week if possible. If F1000 takes longer we could just remove that sentence for now and just link to the talks? :slightly_smiling_face:

Reviewer: Oh, sorry, I thought we had until early Oct. Ok, then this could say
has written up a report for F1000 about the conference, which will be published soon in the [BOSC Collection](https://f1000research.com/collections/bosc)

Author: that sounds good!

> and all the [recordings of the talks are now available on YouTube](https://www.youtube.com/watch?v=AhTZGLnFmh0&list=PLir-OOQiOhXZUNv8DStG8LC1I9-MoUgMh).

Reviewer: ```suggestion
and the [BOSC talk videos are now available on YouTube](https://www.youtube.com/watch?v=AhTZGLnFmh0&list=PLir-OOQiOhXZUNv8DStG8LC1I9-MoUgMh).
```

#### BOSC 2025: Save the date

In 2025, BOSC will take place in Liverpool, UK - as part of the [joint ISMB/ECCB conference](https://www.iscb.org/ismbeccb2025/home). 
The whole event will take place between July 20-24, 2025. 

> The exact dates of BOSC within that range haven't been decided yet, but keep that time already saved in your calendar!

Reviewer: ```suggestion
The exact dates of BOSC within that range haven't been decided yet, but please save those dates in your calendar!
```

To keep up with BOSC news, please follow us on [LinkedIn](https://www.linkedin.com/groups/14344023/), [Bluesky](https://bsky.app/profile/bosc.bsky.social), [Mastodon](https://genomic.social/@BOSC), and [Slack](https://join.slack.com/t/obf-bosc/shared_invite/zt-n5ur1gsj-z2C~69_4lYTFPg5tbWA8Ew)! (We are no longer on Twitter/X; [find out why](https://www.open-bio.org/2023/11/20/leaving-x/).)

### OBF Event Fellowships

Next deadline: December 1, 2024

> The current application period for the OBF Event Fellowship lasts until December 1, 2024. So there is still some [time to get your application submitted](https://forms.gle/3yLg94G1C9nkZkvq5).

Reviewer: ```suggestion
The current application period for the OBF Event Fellowship lasts until December 1, 2024, so there is still time to [get your application submitted](https://forms.gle/3yLg94G1C9nkZkvq5).
```

> These fellowships are ideal to support your virtual or in-person attendance of conferences related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).

Reviewer: Should we consider mentioning that these fellowships are normally limited to $1000? That unfortunately doesn't cover full travel expenses for many meetings these days.

Author: I'd be happy to add that in, @kipkurui & @hzahroh: what do you think?

Reviewer 2: Yes, that's a good idea.
